.github/workflows/ci.yaml

```diff
@@ -100,7 +100,27 @@ jobs:
 
       - name: Python Bindings Tests
         run: just python test
-
+  node-bindings-tests: 
+    name:  Node Binding Tests
+    runs-on: ubuntu-latest-8-cores
```

Comment:
is the mismatch on the cache that we pin this to latest and then in the deploy pin to a specific version?